rogue/src/gui.rs

```diff
@@ -208,29 +207,30 @@ pub fn draw_ui(ecs: &World, ctx: &mut Rltk) {
     let players = ecs.read_storage::<Player>();
     let hunger = ecs.read_storage::<HungerClock>();
     let swim_stamina = ecs.read_storage::<SwimStamina>();
-    for (_player, stats, clock, stamina) in (&players, &combat_stats, &hunger, &swim_stamina).join() {
+    for (_player, stats, clock, stamina) in (&players, &combat_stats, &hunger, &swim_stamina).join()
```

Comment:
Bunch of random formatting changes :disappointed: I'm not sure if this is from tooling version differences or what. I tried to disable the auto-formatting behavior on my text editor, but it looks like a few got through.

I _think_ there's a way to specify the desired dev tooling versions in the project's `Cargo.toml` file. I might play around with that and see if I can get back to consistent formatting. Or run formatting project-wide in a follow-up PR if you're open to that.